### PR TITLE
docs: update local testnet docs title

### PR DESCRIPTION
**Pull request by geekflyer**
### Test Plan

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2019)

### CODE REVIEW

**Comment by rajkaramchedu** [comment]
This should be original title with Validator in it. I made it this way because saying "local testnet" leaves ambiguity, is it with fullnode or validator. This document specifically talks only about Validator. So I suggest we do not make this title change.

**Comment by geekflyer** [comment]
hmm, how about "Run a local testnet with Validator" or something like that? I really would like the term `local testnet` somewhere in the docs title/heading as this is what I would search for.

**Comment by geekflyer** [comment]
The title currently sounds as if we're running a validator that _joins_ an existing local testnet, but really this command/docs are about creating/running a local testnet (which happens to have  validator as a component since every network requires at least 1 validator).

**Comment by rajkaramchedu** [comment]
> hmm, how about "Run a local testnet with Validator" or something like that? I really would like the term `local testnet` somewhere in the docs title/heading as this is what I would search for.

The current title does have "local testnet" in it. But I get your point. We can change it to "Run a Local Testnet with Validator".

File: developer-docs-site/docs/nodes/run-a-local-testnet.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-title: "Validator on Local Testnet"
+title: "Run a Local Testnet"
 slug: "run-a-local-testnet"
 sidebar_position: 9
 ---
 import BlockQuote from "@site/src/components/BlockQuote";
 
-# Validator on Local Testnet
+# Run a Local Testnet
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/nodes/run-a-local-testnet.md:8]: Change both this and the title key in the above front matter to: **Run a Local Testnet with Validator**
(w in with is lowercase)

**Review by rajkaramchedu — APPROVED** [review]
Approving with the suggested change.

**Pull request merged**